### PR TITLE
Updated timeago strings

### PR DESCRIPTION
**Pull request by ranbena**
- [x] Refactor

## Description
1. "a few seconds ago" is too common to be so long. "just now" is better.
2. Sometimes differences in server/client clocks cause the confusing "**in** a few seconds". Now will simply say "just now".
3. Updated the tooltip to ant design.

This also manifests to "Outdated Queries", "Users" list and "Alerts" list.

## Mobile & Desktop Screenshots

Before|After
-------|-------
Screen Shot 2019-07-27 at 16 42 27 | Screen Shot 2019-07-27 at 16 40 41

### CODE REVIEW

File: client/app/components/TimeAgo.jsx [hunk 1]
```diff
@@ -6,6 +6,25 @@ import PropTypes from 'prop-types';
 import { Moment } from '@/components/proptypes';
 import { clientConfig } from '@/services/auth';
 import useForceUpdate from '@/lib/hooks/useForceUpdate';
+import Tooltip from 'antd/lib/tooltip';
+
+moment.updateLocale('en', {
```
Comment (arikfr) [inline client/app/components/TimeAgo.jsx]: As this is something that applies globally it probably should be somewhere else (`app/config`?).

File: client/app/components/TimeAgo.jsx [hunk 2]
```diff
@@ -6,6 +6,25 @@ import PropTypes from 'prop-types';
...
+  relativeTime: {
+    future: '%s',
+    past: '%s',
+    s: 'just now',
```
Comment (arikfr) [inline client/app/components/TimeAgo.jsx]: Did we change any of the other strings or only this one?

Comment (ranbena) [reply client/app/components/TimeAgo.jsx]: Nay.
I experimented with short-handing ("2 minutes ago" to "2m ago") but there's no straightforward way with `moment` to apply it for widgets only.

File: client/app/components/TimeAgo.jsx (hunk 1 above)
Comment (ranbena) [reply client/app/components/TimeAgo.jsx]: 👍 56b6571cca7da6b39192413069615885be957b8b

**Pull request merged**